easybuild/easyconfigs/p/Perl/Perl-5.32.1-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,1798 @@
+        'source_tmpl': 'Config-Simple-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/S/SH/SHERZODR'],
+        'checksums': ['dd9995706f0f9384a15ccffe116c3b6e22f42ba2e58d8f24ed03c4a0e386edb4'],
+    }),
+    ('Business::ISBN', '3.006', {
+        'source_tmpl': 'Business-ISBN-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BD/BDFOY'],
+        'checksums': ['c1fefe68354ffb80cdbd24303ebe684b3b6828df3d5f09b429a09fc4f0919c9a'],
+    }),
+    ('Business::ISBN::Data', '20210112.006', {
+        'source_tmpl': 'Business-ISBN-Data-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BD/BDFOY'],
+        'checksums': ['98c2cfb266b5fdd016989abaa471d9dd4c1d593c508a6f01f66d184d5fee8bae'],
+    }),
+    ('common::sense', '3.75', {
+        'source_tmpl': 'common-sense-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/ML/MLEHMANN'],
+        'checksums': ['a86a1c4ca4f3006d7479064425a09fa5b6689e57261fcb994fe67d061cba0e7e'],
+    }),
+    ('IO::Compress::Bzip2', '2.102', {
+        'source_tmpl': 'IO-Compress-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/P/PM/PMQS'],
+        'checksums': ['d6fa7f9a5beee446452a0fbc43589a0c73fe7e925c075b98628b018048dc72a4'],
+    }),
+    ('JSON::XS', '4.03', {
+        'source_tmpl': 'JSON-XS-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/ML/MLEHMANN'],
+        'checksums': ['515536f45f2fa1a7e88c8824533758d0121d267ab9cb453a1b5887c8a56b9068'],
+    }),
+    ('List::MoreUtils::XS', '0.430', {
+        'source_tmpl': 'List-MoreUtils-XS-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/R/RE/REHSACK'],
+        'checksums': ['e8ce46d57c179eecd8758293e9400ff300aaf20fefe0a9d15b9fe2302b9cb242'],
+    }),
+    ('Authen::NTLM', '1.09', {
+        'source_tmpl': 'NTLM-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/N/NB/NBEBOUT'],
+        'checksums': ['c823e30cda76bc15636e584302c960e2b5eeef9517c2448f7454498893151f85'],
+    }),
+    ('Types::Serialiser', '1.01', {
+        'source_tmpl': 'Types-Serialiser-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/ML/MLEHMANN'],
+        'checksums': ['f8c7173b0914d0e3d957282077b366f0c8c70256715eaef3298ff32b92388a80'],
+    }),
+    ('XML::SAX::Expat', '0.51', {
+        'source_tmpl': 'XML-SAX-Expat-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BJ/BJOERN'],
+        'checksums': ['4c016213d0ce7db2c494e30086b59917b302db8c292dcd21f39deebd9780c83f'],
+    }),
+    ('Inline', '0.86', {
+        'source_tmpl': 'Inline-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/I/IN/INGY'],
+        'checksums': ['510a7de2d011b0db80b0874e8c0f7390010991000ae135cff7474df1e6d51e3a'],
+    }),
```

Comment:
Add since @lexming needed it for 5.32 (not yet merged, but I intend to merge it)

    ('Compress::Raw::Zlib', '2.101', {
        'source_tmpl': 'Compress-Raw-Zlib-%(version)s.tar.gz',
        'source_urls': ['https://cpan.metacpan.org/authors/id/P/PM/PMQS'],
        'checksums': ['9d1b9515e8277c1b007e33fad1fd0f18717d56bf647e3794d61289c45b1aabb2'],
    }),

Reply:
Correction: This was apparently secrelty included in perl, so we have reverted this for perl 5.32.0, so lets not include it here either #12681

Reply:
Done.

Reply:
I need to push it. I will also update the extensions a bit. There were some new releases.